### ISSUE TITLE
[5.3] Query get() method does not return stdClass type

### ISSUE THREAD

**Issue by ted-piotrowski** [laravel/framework#18645]
- Laravel Version: 5.3.31
- PHP Version: PHP 5.6.30-0+deb8u1 AND PHP 7.0.15
- Database Driver & Version: Not relevant

### Description:

It's documented in version 5.1+ [https://laravel.com/docs/5.1/queries#retrieving-results](https://laravel.com/docs/5.1/queries#retrieving-results), but it appears that Laravel 5.3 is returning an `array` type instead of `stdClass` when calling `get()` with a query builder.

```
Laravel 5.4.1+

object(Illuminate\Support\Collection)#1072 (1) {
  ["items":protected]=>
  array(33) {
    [0]=>
    object(stdClass)#1076 (11) 
```

```
Laravel 5.3.31

object(Illuminate\Support\Collection)#951 (1) {
  ["items":protected]=>
  array(35) {
    [0]=>
    array(11) 
```

### Steps To Reproduce:

Run `var_dump(DB::table('table_name'))->get())` with different Laravel versions and compare.

**Comment by deleugpn** [comment]
If this is an issue only in 5.3, it no longer receives bug fixes.

**Comment by Dylan-DPC** [comment]
@ted-piotrowski is this a 5.3 issue? the support period for 5.3 has ended.

**Comment by ted-piotrowski** [comment]
Yes. It's a 5.3 issue. Originally, I thought 5.4 was broken because the return type changed, but it looks like it was actually broken prior to 5.4 and now the correct behavior is implemented starting in 5.4. Feel free to close.
